changelogs/client_server/newsfragments/1508.feature

```diff
@@ -0,0 +1,2 @@
+Document the `m.mentions` property; the `.m.rule.is_user_mention` and `.m.rule.is_room_mention` push rules; and other behaviour from [MSC3952](https://github.com/matrix-org/matrix-spec-proposals/pull/3952).
```

Comment:
The MSC also contains [a section about bridging](https://github.com/matrix-org/matrix-spec-proposals/blob/clokep/intentional-mentions/proposals/3952-intentional-mentions.md#impact-on-bridging) but honestly this isn't too interesting -- I think it mostly has to do with the event format which is covered by this PR.

Do we think we need to include any specific info in the AS spec?

Reply:
Yes, wouldn't bother to call it out.